app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayInteractor.kt

```diff
@@ -30,3 +30,28 @@ interface TabsTrayInteractor {
      */
     fun onDeleteTabs(tabs: Collection<Tab>)
 }
+
+/**
+ * Interactor to be called for any tabs tray user actions.
+ *
+ * @property controller [TabsTrayController] to which user actions can be delegated for actual app update.
+ */
+class DefaultTabsTrayInteractor(
+    private val controller: TabsTrayController
+) : TabsTrayInteractor {
+    override fun onTrayPositionSelected(position: Int, smoothScroll: Boolean) {
+        controller.handleTrayScrollingToPosition(position, smoothScroll)
```

Comment:
There are many situations in which there is one Interactor delegating directly to one Controller just for consistency with  other situations in which an Interactor delegates to other Interactors and/or Controllers.
This could happen when there are multiple (>=2) Views in the same Fragment, each with it's own Interactor implemented to make calls to different other Interactors / Controllers.
https://github.com/mozilla-mobile/fenix/blob/master/docs/architecture-overview.md#interactor